logic-and-algorithm-test/2.py

```diff
@@ -1,18 +1,18 @@
 # Diberikan sebuah string acak, hitunglah berapa jumlah setiap karakter yang ada dalam string
 # tersebut
 def count_characters(input_string):
-    char_count = {}
+    character_count = {}
```

Comment:
The variable name change from 'char_count' to 'character_count' is appropriate and makes the variable's purpose clearer.